### PR TITLE
[Android] PAC does not work with Crosswalk

### PR DESCRIPTION
**Pull request by marcuspridham**
Crosswalk does not work with the Proxy Auto-Config setting set.  There is some changes from https://codereview.chromium.org/742023003 that need to be applied to Crosswalk.  I have verified this change against a Nexus 7 (5.1.1) that has my corporate PAC set.  I also checked with a Nexus 5 (4.4.4)  that has a manual proxy set still works.

BUG=XWALK-5094

### CODE REVIEW

**Comment by rakuco** [comment]
The change looks sane to me. Ping @lincsoon, @halton

File: runtime/browser/runtime_url_request_context_getter.cc [hunk 1]
```diff
@@ -51,6 +51,7 @@
 #include "xwalk/runtime/browser/android/net/url_constants.h"
 #include "xwalk/runtime/browser/android/net/xwalk_url_request_job_factory.h"
 #include "xwalk/runtime/browser/android/xwalk_request_interceptor.h"
+#include "net/proxy/proxy_config_service_android.h"
```
Comment (halton) [inline runtime/browser/runtime_url_request_context_getter.cc]: please order, lgtm with this nit comment.

**Comment by marcuspridham** [comment]
Fixed include order in latest commit. Thanks for the review.

**Comment by halton** [comment]
Thanks for quick rework, will merge it after trybot get green.

**Pull request merged**